osf/models/files.py

```diff
@@ -108,6 +108,8 @@ class BaseFileNode(TypedModel, CommentableMixin, OptionalGuidMixin, Taggable, Ob
     objects = BaseFileNodeManager()
     active = ActiveFileNodeManager()
 
+    mfr_views = models.PositiveIntegerField(default=0)
```

Comment:
np: perhaps better organized above the manager declarations and with the other fields, so they're not obscured, but nothing technically wrong with this

Reply:
Is there a reason you didn't use the `PageCounter` class mentioned in the ticket? I saw there was a discussion in a previous comment but wasn't sure of the resolution as it still seems to be in the ticket.

Reply:
I'd be open to using the PageCounter, but this just seems more intuitive to me. I don't know what the pros of using the PageCounter would be. I think if someone is looking for the view count they'd expect to find it on the file model.

Reply:
The problem with incrementing a field on the file is that the record gets locked every time the file gets viewed, increasing the probabiility of a deadlock. Using `PageCounter` solves this problem and also saves a timestamp for views, which is is important for internal metrics.